include/oneapi/mkl/dft/detail/types_impl.hpp

```diff
@@ -31,7 +31,7 @@ namespace detail {
 
 typedef int DFT_ERROR;
 
-#define DFT_NOTSET -1
+#define DFTI_NOTSET 0 
```

Comment:
I don't think this symbol is helpful

Reply:
what do you mean ? do you prefer the -1 over the 0? this is helpful in cpu/commit.cpp to initialize the status in constructor and set_value_item

Reply:
I don't believe those initializations are necessary